loops_lab.js

```diff
@@ -0,0 +1,128 @@
+// 1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
```

Comment:
Good understanding of loops in general. I would like to see more "for loops" as most of loops in the assignment were while loops. Also I found minor errors in logics within loops for some questions. Keep practicing and keep up the good work!